TicketBooking/src/dip/corrections/BankCard.java

```diff
@@ -0,0 +1,6 @@
+package dip.corrections;
+
+public interface BankCard {
```

Comment:
follow naming conventions, interface name should shart with 'I', for ex: IBankCard